CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # Changelog
 
+
+## [v2.0.0](https://github.com/ably/kafka-connect-ably/tree/v2.0.0) (2022-04-20)
+
+[Full Changelog](https://github.com/ably/kafka-connect-ably/compare/v2.0.0...HEAD)
```

Comment:
This needs to work once the release is done... Looking at the previous release tag, I expect this should be:

```suggestion
[Full Changelog](https://github.com/ably/kafka-connect-ably/compare/v1.0.3...v2.0.0)
```

Reply:
96fbd7d83b59bb617857f7067b9474d24ccf0f59